config/default.json

```diff
@@ -41,5 +41,9 @@
 	},
 	"CTL_TOOLS": {
 		"EXTERNAL_TOOL_MAX_LOGO_SIZE_IN_BYTES": 300000
+	},
+	"ADMIN_API": {
+		"ENABLED": true,
+		"ALLOWED_API_KEYS": "1ab2c3d4e5f61ab2c3d4e5f6"
```

Comment:
Please don't set any default allowed API keys - it should be required from now on if the whole Admin API server feature is enabled, but from the security point of view it would be better to not have any default value specified here in my opinion.